docs/content/latest/explore/multi-region-deployments/row-level-geo-partitioning.md

```diff
@@ -346,7 +393,133 @@ txn_type      | debit
 created_at    | 2020-11-07 21:28:11.056236
 ```
 
-## Step 4. Adding a new geographic location
+## Step 5. Running transactions
+
+So far, we have only been running `SELECT` and [single-row
+transactions](../../architecture/transactions/transactions-overview/#single-row-transactions). With geo-partitioning, there is a new complication introduced for general distributed transactions as shown below.
+
+Let's say we want to run the following transaction:
+```sql
+BEGIN;
+INSERT INTO transactions VALUES (100, 10002, 'EU', 'checking', 400.00, 'debit');
+INSERT INTO transactions VALUES (100, 10003, 'EU', 'checking', 400.00, 'credit');
+COMMIT;
+```
+
+If we attempt to run this while connected to a node in us-west-2:
+```sql
+BEGIN;
+```
+```
+BEGIN
+```
+```sql
+INSERT INTO transactions VALUES (100, 10002, 'EU', 'checking', 400.00, 'debit');
+```
+```
+ERROR:  Illegal state: Nonlocal tablet accessed in local transaction: tablet c5a611afd571455e80450bd553a24a64: . Errors from tablet servers: [Illegal state (yb/client/transaction.cc:284): Nonlocal tablet accessed in local transaction: tablet c5a611afd571455e80450bd553a24a64]
+```
+
+we get an error similar to the above. This is because, since we have created a transaction table for us-west-2, by default, YugabyteDB assumes that we want to run a transaction local to the region (using the transaction status table `system.transactions_us_west_2` which we created in Step 1), but such a transaction cannot modify data outside of us-west-2.
```

Comment:
```suggestion
Because we've created a transaction table for us-west-2, YugabyteDB assumes by default that we want to run a transaction local to that region (using the transaction status table `system.transactions_us_west_2` we created in Step 1), but such a transaction cannot modify data outside of us-west-2.
```